earth_enterprise/src/fusion/autoingest/MiscConfigStorage.idl

```diff
@@ -34,6 +34,9 @@ class MiscConfigStorage {
   uint64       MaxVersionCacheMemorySize = uint64(2415919104);
 
   bool         LimitMemoryUtilization = false;
+
+  // The maximum percentage difference between cache size and dirty map size.
+  float         MaxPurgePercentage = float(.75);
```

Comment:
I think we should set this to a number that makes the behavior identical to previous behavior by default - so 0. That's up for discussion, though.

Reply:
Also, since the meaning of this number has changed we may want to think of a better name. MaxPurgePercent made more sense before you added the `1 - ` in front of it. Now, this number means that at least this percentage of the cache has to not be dirty for us to prune. So maybe `NonDirtyPercentForPrune`? `MinNonDirtyFractionForPrune`? If you want to go crazy you could make it a whole sentence: `MinFractionOfNonDirtyCacheItemsForPruning`. I don't really like any of those options but I think the code will be clearer if the variable name indicates that this is the minimum fraction of the cache that has to be non-dirty for us to prune. You could go with something like `PruneFraction` or `PruneThreshold`, which aren't awful but aren't as descriptive because they don't tell you what it's a threshold of. It also might be easier to name if you reverse the meaning again by taking out the `1 - `, and then it becomes the maxiumum fraction of the cache that can be dirty before we stop pruning, so maybe `MaxCacheDirtyPercentForPruning`?